lib/auth/github.go

```diff
@@ -473,6 +474,23 @@ func (c *githubAPIClient) get(url string) ([]byte, string, error) {
 	return bytes, wls.NextPage, nil
 }
 
+// getWebSessionTTL returns web session TTL
```

Comment:
the significance of this function is not clear to me.

Reply:
AdjustSessionTTL with max ttl as a parameter will return similar results, so this is a code duplication. Besides, I'm not sure that this is the right behavior for github SSO specifically.